src/Magento/FunctionalTestingFramework/Util/DocGenerator.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\FunctionalTestingFramework\Util;
+
+use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
+use Magento\FunctionalTestingFramework\Test\Handlers\ActionGroupObjectHandler;
+use Magento\FunctionalTestingFramework\Test\Objects\ActionGroupObject;
+use Magento\FunctionalTestingFramework\Test\Objects\TestObject;
+
+/**
+ * Class TestGenerator
+ * @SuppressWarnings(PHPMD)
+ */
+class DocGenerator
+{
+    const DEFAULT_OUTPUT_DIR =
+        PROJECT_ROOT .
+        DIRECTORY_SEPARATOR .
+        "dev" .
+        DIRECTORY_SEPARATOR .
+        "tests" .
+        DIRECTORY_SEPARATOR .
+        "docs";
+    const DOC_NAME = "documentation.md";
+    const ANNOTATION_MODULE = "";
+    const ANNOTATION_PAGE = "page";
+    const ANNOTATION_DESCRIPTION = "description";
```

Comment:
Does it make sense to keep these `const` in `ActionGroupObject` instead? I'm just thinking of maintenance cost if we change schema or add more annotations, where does it make sense to keep these `const`s that correspond to schema changes.

Reply:
I'm not sure on this one. this seems to be different per area of the framework.

Reply:
Well, mentally I think `actionGroup schema` => `actionGroup class` if I needed to take a look at any constants. Similarly in `actionObject` we have `const ACTION_ATTRIBUTE_TIMEZONE = 'timezone'` for the attribute that lives in the schema.

I don't feel all that strongly on this one though, if you disagree then feel free to ignore this.